benchmarks/benchmark/tools/dlio/modules/parallelstore_storage/dataloader-job-rendered.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: data-loader-job
+  namespace: default
+spec:
+  backoffLimit: 0
+  template:
+    metadata:
+      name: data-loader-job
+      annotations:
+        gke-parallelstore/volumes: "true"
+        gke-parallelstore/cpu-limit: 1000m
+        gke-parallelstore/memory-limit: 5000Mi
```

Comment:
Same here, do we want to use var here?

Reply:
oh, sorry, I didn't notice that this file is auto generated, then maybe remove this file?

Reply:
done. removed.